app/decorators/models/course_decorator.rb

```diff
@@ -99,6 +102,19 @@ def get_relevant_student_alerts_count(student)
     ).payload.size
   end
 
+  def set_course_start_time_from_school
+    unless self.start_at.nil?
```

Comment:
Trying to understand... why does the `start_at` column only get set if it's not `nil` when the callback runs?

Reply:
When creating a course through import, which is what a majority of the users do, the dates/time are set already and the time needs to take on the account level start time. When creating a course manually through the UI, the course start_at is nil. They want it to have to be set manually as opposed to importing from account level.

Reply:
Makes sense. Thanks for the explanation!